runtime/nls/j9cl/j9jcl.nls

```diff
@@ -407,3 +407,18 @@ J9NLS_JCL_OOM_NEW_GLOBAL_REF.explanation=The JVM was not able to allocate the na
 J9NLS_JCL_OOM_NEW_GLOBAL_REF.system_action=The JVM will exit unless the OutOfMemoryError is caught and recovered from.
 J9NLS_JCL_OOM_NEW_GLOBAL_REF.user_response=Ensure the process has enough native memory to run.
 # END NON-TRANSLATABLE
+
+# Message for IllegalArgumentException if host class and anonymous class are in different packages
+# argument 1 is the host class
+# argument 2 is the anonymous class
+
+J9NLS_JCL_HOSTCLASS_ANONCLASS_DIFFERENT_PACKAGES=Host class %2$.*1$s and anonymous class %4$.*3$s are in different packages
```

Comment:
It seems createErrorMessage() only prints the package name for the anonymous class, this error message can be re-worded to read better.

Reply:
it should print host and anon class names with package name. example - ` java.lang.IllegalArgumentException: Host class java/lang/Object and anonymous class test/DummyClass are in different packages`.

Reply:
Then I don't understand this code in createErrorMessage(). It appears to strip the class name, and would include the trailing '/'

```
for (; anonClassNameLength >= 0; anonClassNameLength--) {
			if (anonClassNameData[anonClassNameLength] == '/') {
```

Reply:
anonymous class name has trailing digits. example - `test/DummyClass/00000000442F098`. the above code removes `/00000000442F098` from anonymous class name. i will add a comment in the code.

Reply:
Ok. There is a comment already, but adding an example name would be helpful

Reply:
added an example.